src/index.html

```diff
@@ -6,19 +6,26 @@
 {% block content %}
 <main class="content content__2-1 content__bleedbar" id="main" role="main">
 
-    <section class="content_hero">
+    <div class="content-hero">
 
       <!-- BACKGROUND IMAGE FOR THE HERO -->
-      <div id="hero-bg-image"></div>
 
        <!-- HERO -->
-      <div class="content_wrapper content_no-sidebar">
-          <section class="content_main">
-            <h1>Owning a Home</h1>
-            <h2>We're here to help you with your home-buying process. Learn what to expect and what questions to ask, and get tools and resources to help you make decisions.</h2>
-          </section>
-      </div>
-    </section>
+      <section class="page-intro wrapper">
```

Comment:
why did you replace `content_wrapper` here? it should help with keeping whitespace and layout consistent between the hero and the columns after.